scripts/collect_images_from_gaze_pose.py

```diff
@@ -0,0 +1,204 @@
+"""
+Add in $ROOT/scripts/
+Create $ROOT/data/RGB directory
+
+To run:
+- python3 scripts/collect_ig.py 192.168.80.3 --to-depth --gaze-x 0.75 --gaze-y 0.0 --gaze-z 0.4 --gaze-roll 0.0 --gaze-yaw 0.0
+
+default --gaze-pose: 
+0.75, 0.0, 0.4, np.pi/2, 0.0, 0
+"""
+
+import argparse
+import sys
+import cv2
+import time
+import numpy as np
+
+import bosdyn.client.estop
+import bosdyn.client.lease
+from bosdyn.client.lease import LeaseClient
+from bosdyn.api import estop_pb2
+from bosdyn.client.estop import EstopClient
+from bosdyn.client.image import ImageClient
+from bosdyn.client.manipulation_api_client import ManipulationApiClient
+from bosdyn.client.robot_command import RobotCommandClient, blocking_stand
+from bosdyn.client.robot_state import RobotStateClient
+from bosdyn.client.ray_cast import RayCastClient
+from bosdyn.client.robot_command import (RobotCommandBuilder, RobotCommandClient,
+                                         block_until_arm_arrives, blocking_stand)
+
+from conq.hand_motion import hand_pose_cmd
+from conq.manipulation import blocking_arm_command
+
+from pathlib import Path
+from PIL import Image
+
+import bosdyn.client
+import bosdyn.client.util
+from bosdyn.client.image import ImageClient
+from conq.cameras_utils import get_color_img, RGB_SOURCES, DEPTH_SOURCES
+from conq.clients import Clients
+
+
+# FIXME: To be ignored during push (Stanley's code to be used only for image collection)
+def move_to(clients: Clients, pose, duration = 1, follow=False):
+    """
+    Move the arm to a pose relative to the body
+
+    Args:
+        clients: Clients
+        x: x position in meters in front of the body center
+        y: y position in meters to the left of the body center
+        z: z position in meters above the body center
+        roll: roll in radians
+        pitch: pitch in radians
+        yaw: yaw in radians
+        duration: duration in seconds
+
+    Adapted from conq.manipulation:hand_pose_cmd
+    """
+    
+    try:
+        print("Arm command received")
+        x,y,z,pitch,roll,yaw = pose
+        arm_cmd = hand_pose_cmd(clients, x,y,z,roll,pitch,yaw,duration)
+        
+        blocking_arm_command(clients, arm_cmd)
+        print("Arm command done")
+        return True
+    except Exception as e:
+        print(e)
+        return False
+    
+
+def open_gripper(clients: Clients):
+    try:
+        clients.command.robot_command(RobotCommandBuilder.claw_gripper_open_command())
+        time.sleep(1)  # FIXME: how to block on a gripper command?
+        return True
+    except: return False
```

Comment:
newline between `except:` and `return`

Reply:
I have deleted this file from this branch, it's not needed here. I'll correct(complete?) it and raise a new PR for this thing.